tce/src/tools/Environment.cc

```diff
@@ -704,6 +704,22 @@ Environment::hdbPaths(bool libraryPathsOnly) {
     return paths;
 }
 
+std::string
+Environment::shortHDBPath(std::string hdbPath) {
+    if (StringTools::startsWith(hdbPath, TCE_SRC_ROOT)) {
```

Comment:
I think this check needs TCE_SRC_ROOT + "/" otherwise it matches $(TCE_SRC_ROOT)foobar too?